.rspec

```diff
@@ -1,3 +1,4 @@
+--require manageiq/spec/spec_helper
```

Comment:
Although this seems very convenient, this is an expensive require and precisely the kind of thing I was aiming to avoid in #22. As you can see there, I've moved any expensive requires to the `spec/rails_helper` file, which is only required by integration tests. Although this is a bit of a departure from the way we test in every other repo, it is a good convention introduced in (I believe) rspec-rails v3, and means we can get near-instant feedback on small, unit-style tests as opposed to the typical 8-10 seconds it takes just to load and boot rails.

Reply:
I agree, this can be removed. Plugins can and should certainly have some flexibility about the dependencies on their own test types, this isn't something that's necessarily always convenient to have everything loaded for every test for every plugin. 

Here we can utilize tests that might only have to do with the type system and nothing to do with business logic, and those tests can be lightning fast without this global require.

Note this doesn't mean that business logic tests won't have ManageIQ automatically loaded for contributors. Just the mechanism is different (we'll load it through metadata/dir, not this dotfile). This is totally invisible to anyone who doesn't actually care.